src/rf/apps/home/tests.py

```diff
@@ -72,6 +72,19 @@ def setup_models(self):
             layer = self.make_layer(layer_name, is_public=False)
             self.save_layer(layer, user)
 
+    def make_many_layers(self):
+        """
+        Create 30 public layers.
+        """
+        for username in self.usernames:
+            self.client.login(username=username, password=username)
+            user = self.user_models[username]
+
+            for i in range(0, 15):
```

Comment:
Thanks, missed that outer loop. A more accurate comment might be "Create 15 layers for each user"?